src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -57,75 +65,116 @@ def add_model_args(parser):
-        "--batch-size",
         type=int,
         default=1,
         required=False,
-        help="The batch size / concurrency to benchmark. (Default: 1)",
+        help="The batch size to benchmark. The default value is 1.",
     )
-    parser.add_argument(
-        "--input-length",
+    exclusive_group.add_argument(
+        "--concurrency",
         type=int,
-        default=128,
         required=False,
-        help="The input length (tokens) to use for benchmarking LLMs. (Default: 128)",
+        help="Sets the concurrency value to benchmark.",
     )
     parser.add_argument(
-        "--output-length",
+        "--max-threads",
         type=int,
-        default=128,
+        default=16,
         required=False,
-        help="The output length (tokens) to use for benchmarking LLMs. (Default: 128)",
+        help="Sets the maximum number of threads that will be "
+        "created for providing desired concurrency or request rate. "
+        "However, when running in synchronous mode,this value will be ignored. "
+        "The default value is 16.",
     )
-
-
-def add_endpoint_args(parser):
+    # TODO: necessary?
+    # parser.add_argument(
+    #     "--output-length",
+    #     type=int,
+    #     default=128,
+    #     required=False,
+    #     help="The output length (tokens) to use for benchmarking LLMs. (Default: 128)",
+    # )
     parser.add_argument(
-        "--url",
-        type=str,
-        default="localhost:8001",
+        "--profile-export-file",
+        type=Path,
+        default="profile_export.json",
+        help="Specifies the path where the profile export will be "
+        "generated. By default, the profile export will not be "
+        "generated.",
+    )
+    exclusive_group.add_argument(
+        "--request-rate",
+        type=float,
         required=False,
-        help="URL of the endpoint to target for benchmarking.",
+        help="Sets the request rate for the load generated by PA. ",
     )
     parser.add_argument(
-        "--provider",
+        "--service-kind",
         type=str,
         choices=["triton", "openai"],
+        default="triton",
         required=False,
-        help="Provider format/schema to use for benchmarking.",
+        help="Describes the kind of service perf_analyzer will "
+        'generate load for. The options are "triton" and '
+        '"openai". The default value is "triton".',
     )
-
-
-def add_dataset_args(parser):
     parser.add_argument(
-        "--dataset",
-        type=str,
-        default="OpenOrca",
-        choices=["OpenOrca", "cnn_dailymail"],
+        "--streaming",
+        action="store_true",
+        required=False,
+        help=f"Enables the use of the streaming API.",
+    )
+    parser.add_argument(
+        "--version",
+        action="store_true",
         required=False,
-        help="HuggingFace dataset to use for the benchmark.",
+        help=f"Prints the version and exits. By default, it is set false.",
```

Comment:
Don't think you need to mention the default for `store_true` types of args
```suggestion
        help=f"Prints the version and exits.",
```